etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiOrderConverterHelper.java

```diff
@@ -0,0 +1,46 @@
+package gov.hhs.cdc.trustedintermediary.external.hapi;
+
+import java.util.List;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.Patient;
+
+/** Helper class with transformation methods that take an order Bundle and modifies it */
+public class HapiOrderConverterHelper {
+
+    private static final Coding OML_CODING =
+            new Coding(
+                    "http://terminology.hl7.org/CodeSystem/v2-0003",
+                    "O21",
+                    "OML - Laboratory order");
+
+    private static final List<Coding> CODING_LIST =
+            List.of(
+                    new Coding(
+                            "http://terminology.hl7.org/CodeSystem/v3-RoleCode", "MTH", "mother"));
+
+    public static void convertToOmlOrder(Bundle order) {
```

Comment:
Could we make these methods more generic? Like ConvertToOrderType

Reply:
That's a great idea. As I wrote on a response to Jorge, I think we should refactor this helper in a separate PR